build_info/gitea.control

```diff
@@ -0,0 +1,9 @@
+Package: gitea
+Version: @DEB_GITEA_V@
+Architecture: @DEB_ARCH@
+Maintainer: @DEB_MAINTAINER@
+Section: Data_Storage
+Priority: optional
+Depends: git, libpam2
+Pre-Depends: system-cmds (>= 854.40.2-2)
```

Comment:
While the order doesn't matter, it makes more sense to write Pre-Depends first followed by Depends.